docs/model_configuration.md

```diff
@@ -296,8 +296,9 @@ and [Onnxruntime](https://github.com/triton-inference-server/onnxruntime_backend
 backends as examples of how to acheive this. Currently, only
 [inputs, outputs](#inputs-and-outputs), [max_batch_size](#maximum-batch-size)
 and [dynamic batching](#dynamic-batcher) settings can be populated by
-backend. You would still need to provide a config.pbtxt with
-`backend` field for these custom backends.
+backend. For custom backends, your config.pbtxt file must
+include a `backend` field or your model name must be in the
+form `<model_name>.<backend_name>`.
```

Comment:
>`<model_name>.<backend_name>`.

Should be "model.<backend_name>" the way you have coded it.

Reply:
We might be talking about two different things here. To make it [consistent with what we expect for the other frameworks](https://github.com/triton-inference-server/server/blob/main/docs/model_repository.md#model-files), we'd want the model file to be named model.<backend>. However, the model name is pulled from the model directory unless the user creates a model mapping to override it (directory->model_name).

So, for example, a model `add_sub` would be in the subdirectory `add_sub` in the model repository. But in `model_repository/add_sub/1`, there would be a model file named `model.<backend>`.

Would you prefer it done or documented a different way? We don't need to set the default model filename for a custom backend, but I thought it would be a nice touch to be consistent with the other backends, so users can use that field if they so desire.

Reply:
I think that you're right, I was thinking of something different. Thank you for clarifying, I think the way you have it documented should be fine.